spec/spec_helper.rb

```diff
@@ -1,18 +1,11 @@
-# This file was generated by the `rspec --init` command. Conventionally, all
-# specs live under a `spec` directory, which RSpec adds to the `$LOAD_PATH`.
-# The generated `.rspec` file contains `--require spec_helper` which will cause
-# this file to always be loaded, without a need to explicitly require it in any
-# files.
-#
-# Given that it is always loaded, you are encouraged to keep this file as
-# light-weight as possible. Requiring heavyweight dependencies from this file
-# will add to the boot time of your test suite on EVERY test run, even for an
-# individual file that may not need all of that loaded. Instead, consider making
-# a separate helper file that requires the additional dependencies and performs
-# the additional setup, and require it from the spec files that actually need
-# it.
-#
-# See http://rubydoc.info/gems/rspec-core/RSpec/Core/Configuration
+# frozen_string_literal: true
+
+require_relative "../app/pb_kits/playbook/props.rb"
+Dir[File.expand_path("../app/pb_kits/playbook/props/*.rb", __dir__)].each { |f| require f }
```

Comment:
This could go right into `props.rb`, actually

Reply:
Moved it there in a followup commit.

---

As we add tests to kits that include the `Props` module, each spec file can require the one file they need at a time. Once all our converted, we can erase all the single line require statements from the tests, remove the `PbKit::Base` class, and add `Dir[File.expand_path("../app/pb_kits/playbook/**/*.rb", __dir__)].each { |f| require f }` into the spec helper.